src/resources/third_party_externs.js

```diff
@@ -127,4 +127,10 @@ ClassNamespace.constObj = {};
 /**
  * @type {number}
  */
-ClassNamespace.constObj.num;
\ No newline at end of file
+ClassNamespace.constObj.num;
```

Comment:
Please create a second test file for this, this is getting way too long. OK to do in a separate PR.